stories/topics/con-gcp-setup-after-depoyment.adoc

```diff
@@ -0,0 +1,10 @@
+[id="con-gcp-setup-after-deployment"]
+
+= Setting up monitoring and logging after deployment
+
+You can start or stop the logging and monitoring after the deployment by using the `gcp_setup_logging_monitoring` playbook available from registry.redhat.com.
```

Comment:
This would need to be the same URL we are using for all other playbooks to point to the correct $IMAGE.  Right now we have been using quay.io/ansible-on-clouds/ansible-on-clouds-ops-dev:2.3-latest-rh_inc since we do not have a finalized downstream image name/tag.  I would assume this would need to be a variable or point to a section that has the correct image name/tag (once we know it)